crates/bin/c8y-device-management/src/main.rs

```diff
@@ -35,7 +36,12 @@ async fn main() -> anyhow::Result<()> {
 
     // Create actor instances
     let mqtt_config = mqtt_config(&tedge_config)?;
-    let mut mqtt_actor = MqttActorBuilder::new(mqtt_config.clone().with_session_name(PLUGIN_NAME));
+    let mut mqtt_actor = MqttActorBuilder::new(
+        mqtt_config
+            .clone()
+            .with_session_name(PLUGIN_NAME)
+            .with_last_will_message(health_status_down_message(PLUGIN_NAME)),
```

Comment:
The init message is missing and must be added.

One way, to avoid this kind of miss, is to add a `set_init_and_last_will` method to the health actor builder to update the MQTT config with the init and last-will messages.

Reply:
Fixed